### DISCUSSION TITLE
Open all child nodes hyperlinks

### DISCUSSION THREAD

**Discussion by olazzzzz** [freeplane/freeplane#1867] [Ideas]
Hi everyone,

I would like to have a script (or maybe this is already built-in, and I'm unaware) that can open all child node hyperlinks.
So by clicking on A, it would open all hyperlinks (webpages) in A1, A2, A3 and so on.
Is this possible to accomplish?
Thanks in advance for any help.

**Comment by Illegit** [comment]
[OpenLinkOfAllSelectedNodes.zip](https://github.com/user-attachments/files/15837211/OpenLinkOfAllSelectedNodes.zip)
Hi, try this script. It opens links of all of the selected nodes.

**Comment by Illegit** [comment]
You have to select the nodes and run the script.

**Comment by olazzzzz** [comment]
Thank you very much @Illegit , precisely what I needed!
